docs/website/data/integrations.yaml

```diff
@@ -146,6 +146,21 @@ integrations:
     inventors:
       - fig
 
+  fiber:
+    title: fiber
+    description: |
+      Fiber is an Express inspired web framework built on top of Fasthttp, the fastest HTTP engine for Go.
+      Designed to ease things up for fast development with zero memory allocation and performance in mind.
+      With Open Policy Agent integration, you can run your Rego policies as part of the request lifecycle in the middleware.
+    labels:
+      category: application
+      layer: library
+    software:
+      - golang
+      - fiber
```

Comment:
You'll need to add the corresponding items to the "software" object further down in this file.

Reply:
Oh, sorry I missed that part 🙏